### PR TITLE
Add application gateway

### PR DESCRIPTION
**Pull request by rchessa**

### CODE REVIEW

File: westeurope/api_management/terraform.tfvars [hunk 1]
```diff
@@ -15,20 +15,12 @@ terragrunt = {
...
-
-publisher_email = "io-dev@agid.gov.it"
-
+# APIM module variables
+apim_name                 = "01"
+virtualNetworkType        = "internal"
+vnet_name                 = "common"
+subnet_name               = "apim"
+publisher_name            = "Digital Citizenship"
+publisher_email           = "io-dev@agid.gov.it"
```
[the remark this answers is not in the document]
Comment (l-pinna) [reply westeurope/api_management/terraform.tfvars]: Yes, sure.
what about publisher_name ? Should be something like "IO Italia"

File: westeurope/api_management/terraform.tfvars [hunk 2]
```diff
@@ -15,20 +15,12 @@ terragrunt = {
...
 notification_sender_email = "io-dev@agid.gov.it"
```
[the remark this answers is not in the document]
Comment (l-pinna) [reply westeurope/api_management/terraform.tfvars]: ack

File: westeurope/api_management/terraform.tfvars (hunk 1 above)
[the remark this answers is not in the document]
Comment (LucaPrete) [reply westeurope/api_management/terraform.tfvars]: So, publisher name should be IO Italia. publisher email: io-alerts@teamdigitale.governo.it

File: westeurope/api_management/terraform.tfvars (hunk 2 above)
[the remark this answers is not in the document]
Comment (LucaPrete) [reply westeurope/api_management/terraform.tfvars]: 👌

**Pull request merged**